terraform/modules/redis/variables.tf

```
@@ -0,0 +1,27 @@
+variable "cluster_name" {
+  description = "name of the cluster"
+  type        = string
+  default     = "shared"
+}
+
+variable "internal_domain_name" {
+  description = "The internal root root domain name"
```

"root" is repeated here. Might as well just replace it the better description from above:

```suggestion
description = "Domain in which to create DNS records for private resources. For example, test.govuk-internal.digital"
```

---

will do